lib/eunit/src/eunit_surefire.erl

```diff
@@ -203,9 +203,9 @@ handle_cancel(test, Data, St) ->
 		     testcases=[TestCase|TestSuite#testsuite.testcases] },
     St#state{testsuites=store_suite(NewTestSuite, TestSuites)}.
 
-format_name({Module, Function, Arity}, Line) ->
-    lists:flatten([atom_to_list(Module), ":", atom_to_list(Function), "/",
-		   integer_to_list(Arity), "_", integer_to_list(Line)]).
+format_name({Module, Function, _Arity}, Line) ->
+    lists:flatten([atom_to_list(Module), ":", integer_to_list(Line), " ",
+                   atom_to_list(Function)]).
```

Comment:
Shouldn't this be like below? We're usually using Module:Function everywhere. 
```
    lists:flatten([atom_to_list(Module), ":", atom_to_list(Function), " ",
                   integer_to_list(Line)]).
````